NethermindNode.Tests/NethermindNode.Tests.csproj

```
@@ -38,12 +38,27 @@
     </PackageReference>
     <PackageReference Include="NunitXml.TestLogger" Version="3.0.131" />
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="8.0.0" />
-	<None Include="..\config.json" Link="config.json" CopyToOutputDirectory="Always" />
+    <None Include="..\config.json" Link="config.json" CopyToOutputDirectory="Always" />
   </ItemGroup>
 
+  <Choose>
+    <When Condition=" $(DefineConstants.Contains('INCLUDE_SUBMODULES')) ">
+      <ItemGroup>
+        <ProjectReference Include="..\NethermindNode.SedgeFuzzer\NethermindNode.SedgeFuzzer.csproj" />
+        <ProjectReference Include="..\nethermind\src\Nethermind\Nethermind.Runner\Nethermind.Runner.csproj" />
+      </ItemGroup>
+    </When>
+    <Otherwise>
+      <ItemGroup>
+        <ProjectReference Include="..\NethermindNode.SedgeFuzzer\NethermindNode.SedgeFuzzer.csproj" />
+      </ItemGroup>
+    </Otherwise>
+  </Choose>
```

Not sure how this syntax works, but can we do something like this instead? (not sure if indentation works well in suggestion)


```suggestion
      <ItemGroup>
        <ProjectReference Include="..\NethermindNode.SedgeFuzzer\NethermindNode.SedgeFuzzer.csproj" />
          <Choose>
            <When Condition=" $(DefineConstants.Contains('INCLUDE_SUBMODULES')) ">
              <ProjectReference Include="..\nethermind\src\Nethermind\Nethermind.Runner\Nethermind.Runner.csproj" />
            </When>
          </Choose>
      </ItemGroup>
```

---

AFAIK it is not possible - can be only on higher level than item group

https://learn.microsoft.com/en-us/visualstudio/msbuild/choose-element-msbuild?view=vs-2022

---

@brbrr found much better way for doing that - one liner :)

---

Noice!